Adding UDFs to Fusion readiness (#8467)
fishtown-analytics/docs.getdbt.com: website/docs/docs/fusion/fusion-readiness.md

### Validate and upgrade your dbt packages

The most commonly used dbt Labs managed packages (such as `dbt_utils` and `dbt_project_evaluator`) are already compatible with <Constant name="fusion" />, as are a large number of external and community packages. Review [the dbt package hub](https://hub.getdbt.com) to see verified <Constant name="fusion" />-compatible packages by checking that the `require-dbt-version` configuration includes `2.0.0` or higher. Refer to [package support](/docs/fusion/supported-features#package-support) for more information.

- [ ] Make sure that all of your packages are upgraded to the most recent version, many of which contain enhancements to support <Constant name="fusion" />. 
- [ ] Check package repositories to make sure they're compatible with <Constant name="fusion" />. If a package you use is not yet compatible, we recommend opening an issue with the maintainer, making the contribution yourself, or removing the package temporarily before you upgrade.

### Validate user-defined functions

> Validate that all user-defined functions (UDFs) you use are supported. If a function isn't supported (results in a deprecation warning `dbt0209: No function <function name>`):

Reviewer: ```suggestion
Check that all user-defined functions (UDFs) in your project are supported by <Constant name="fusion" />. Unsupported functions trigger a deprecation warning: `dbt0209: No function <function name>`.

If you see this warning:
```

Author: See below
```suggestion
Check that all user-defined functions (UDFs) in your project are supported by <Constant name="fusion" />. Unsupported functions trigger a deprecation warning: `dbt0209: No function <function name>`.

If you see this warning for:
```

> - [ ] **Custom UDF:** Create it as a [native dbt UDF](/docs/build/udfs#defining-udfs-in-dbt) for the full <Constant name="fusion" /> experience, including `static_analysis: on`.

Reviewer: ```suggestion
- [ ] **For custom UDFs**: Redefine them as [native dbt UDFs](/docs/build/udfs#defining-udfs-in-dbt) to get the full <Constant name="fusion" /> experience with `static_analysis: on`.
```

Author: I'm not sure that "redefine" is the term we want here based on the presentations internally, maybe
```suggestion
- [ ] **For custom UDFs:** Recreate it as a [native dbt UDF](/docs/build/udfs#defining-udfs-in-dbt) for the full <Constant name="fusion" /> experience with `static_analysis: on`.
```

> - [ ] **Warehouse-native functions:** Submit a [GitHub issue](https://github.com/dbt-labs/dbt-fusion). You can continue with `static_analysis: off` when on <Constant name="fusion" /> as an interim solution. 

Reviewer: ```suggestion
- [ ] **For Warehouse-native functions:** Submit a [GitHub issue](https://github.com/dbt-labs/dbt-fusion). As a workaround, you can set `static_analysis: off` when running on <Constant name="fusion" />. 
```
